common/src/main/java/com/wynntils/services/mapdata/providers/json/JsonProvider.java

```diff
@@ -161,6 +161,11 @@ public void onChange(Consumer<MapDataProvidedType> callback) {
         // reload of the file, so we do not need to register callbacks.
     }
 
+    @Override
+    public void reloadData() {
+        // To be implemented if needed
```

Comment:
I think we should implement it, if /reload is supposed to do anything. I guess it means we need to let the json provider somehow have a hook back to how it was loaded/created in the first place.

I am also fine with that you do not fix it right now, but then please turn this into a `FIXME`.

Reply:
I did not do this because I was not sure how to. I'll definitely fix it when it comes to that.